Source: pullmoll
Upstream: no (should be reported)
Reason: needs binary and (&) to test for bit mask

--- a/fsck_hfs.tproj/dfalib/SBTree.c	2012-02-01 18:17:19.000000000 +0100
+++ b/fsck_hfs.tproj/dfalib/SBTree.c	2019-05-26 12:45:16.688514031 +0200
@@ -502,7 +502,7 @@
 			{
 				if ( recordSize != sizeof(HFSPlusCatalogFolder) )
 					return false;
-				if ( (catalogRecord->hfsPlusFolder.flags && (kHFSFileLockedMask | kHFSThreadExistsMask)) != 0 )
+				if ( (catalogRecord->hfsPlusFolder.flags & (kHFSFileLockedMask | kHFSThreadExistsMask)) != 0 )
 					return false;
 					
 				cNodeID = catalogRecord->hfsPlusFolder.folderID;
